modules/API/external/external_api.js

```diff
@@ -429,11 +434,17 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
             const userID = data.id;
 
             switch (name) {
-            case 'video-conference-joined':
+            case 'video-conference-joined': {
+                if (typeof this._tmpE2EEKey !== 'undefined') {
+                    this.executeCommand(commands.e2eeKey, this._tmpE2EEKey);
```

Comment:
I don't see how this is handled on the jitsi-meet side. We are not listening for the command in the API.js. Am I missing something?

Reply:
Oh sorry. I saw it now :) my bad!

but why not include it in the URL?

Reply:
Because the URL is currently leaned up beefore he conference is created so we'd lose the parameter, alas. I may move it in the future if that is refactored, but I had no other choie right now.

Reply:
Wait. You are anyway storing it in redux. So how this is a problem? When creating the conference you can read it from redux and pass it. Am I missing something?